packages/components/Tabs/src/TabsItem.tsx

```diff
@@ -64,7 +67,7 @@ export const TabsItem = compose<TabsItemType>({
       info: {
         ...pressable.state,
         selected: info.selectedKey === userProps.itemKey,
-        icon: !!icon,
+        icon: !!icon && Platform.OS !== 'windows',
```

Comment:
perhaps add a comment + a bug number describing how windows doesn't support SVG icons yet? I assume that's why the check is here

Reply:
Good idea! Yeah, that is the reason of the check.

Reply:
I was not able to find a bug number, but I found the [Status](https://github.com/microsoft/fluentui-react-native/wiki/Status-of-Controls) of the Control. Currently, Icons are on backlog for windows.

Reply:
I meant to open a bug on our GitHub issue tracker, sorry. Should have been clear.

Reply:
Since there is already an Icon Control for the other platforms, would it be a "Feature Request" or a "New Component"?

Reply:
Bug created: #935